dataland-frontend/tests/e2e/specs/lksg/DataIntegrity.ts

```diff
@@ -91,12 +92,14 @@ describeIf(
             const reportingYear: number = +reportingYearAsString;
             reportingYearOfNewLksgDataSet = reportingYear + 1;
           }
-          const newLksgDataSet = generateLksgData(
-            reportingYearOfNewLksgDataSet.toString() + dateAndMonthOfAdditionallyUploadedLksgDataSets
+          const reportingDateOfNewLksgDataSet =
+            reportingYearOfNewLksgDataSet.toString() + dateAndMonthOfAdditionallyUploadedLksgDataSets;
+          const newLksgDataSet = generateLksgData(reportingDateOfNewLksgDataSet);
+          return uploadOneLksgDatasetViaApi(token, companyId, reportingDateOfNewLksgDataSet, newLksgDataSet).then(
```

Comment:
I would expect the reporting period to be rather the year than the full date

Reply:
You are right. This must be an old state from when Julia asked us to have dates as reporting periods for lksg datasets.

I fixed it now to be a year again. I also renamed some things here to be more precise.